backend/.editorconfig

```diff
@@ -76,7 +76,7 @@ dotnet_diagnostic.S4457.severity = error
 # S3442 "abstract" classes should not have "public" constructors
 dotnet_diagnostic.S3442.severity = error
 # S125 Remove commented out code
-dotnet_diagnostic.S125.severity = error
+dotnet_diagnostic.S125.severity = warning
```

Comment:
Haha, nice.

Reply:
I agree with this, it was kinda annoying during dev.

Reply:
yeah, I meant to remove it earlier...